libraries/abstractions/secure_sockets/lwip/iot_secure_sockets.c

```diff
@@ -464,6 +464,11 @@ int32_t SOCKETS_Send( Socket_t xSocket,
         return SOCKETS_EINVAL;
     }
 
+    if( ( ctx->status & SS_STATUS_CONNECTED ) != SS_STATUS_CONNECTED )
```

Comment:
Two issues:

1. `ctx` has not been assigned yet, that doesn't happen until line 472.
1. The commit message does not describe this change; `xSocket != SOCKETS_INVALID_SOCKET` was guaranteed by line 457.  If this is a change we want to make, it should be in a separate commit/PR with a message like "Check for disconnected socket in SOCKETS_Send."